src/js/core/RowManager.js

```diff
@@ -141,7 +141,7 @@ export default class RowManager extends CoreFeature{
 				this.dispatch("scroll-vertical", top, topDir);
 				this.dispatchExternal("scrollVertical", top, topDir);
 			}
-		});
+		}, {passive: true});
```

Comment:
I have changed to a passive event listener here for scroll performance.
I am not sure if you want a polyfill or feature check in unsupported browsers?

https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#browser_compatibility

Reply:
This cannot be a passive event listener because there are touch events listened to by the interaction manager that can trigger interactions that will result in it being a non passive option

Reply:
sorry about that I did not notice, I have reverted the change.